packages/client/lib/net/protocol/snapprotocol.ts

```diff
@@ -239,6 +286,7 @@ export class SnapProtocol extends Protocol {
     super(options)
 
     this.chain = options.chain
+    this.convertSlimBody = options.convertSlimBody
```

Comment:
This defaults to `undefined` / `false`?

Reply:
yes, i wasn't sure what we want default to be, so chose that this be passed as explicit flag to make any conversions